code/game/objects/items/weapons/inducer.dm

```diff
@@ -0,0 +1,190 @@
+	if(!cell.charge)
+		to_chat(user, "<span class='warning'>The [src]'s battery is dead!</span>")
+		return
+
+	var/obj/item/weapon/stock_parts/cell/C = W.get_cell()
+	if(C)
+		if(C.charge == C.maxcharge)
+			to_chat(user, "<span class='notice'>The [W] is fully charged!</span>")
+			return
+		user.visible_message("[user] starts recharging the [W] with \the [src]",\
+							"<span class='notice'>You start recharging the [W] with \the [src]</span>")
+		if(do_after(user, 60, target = W, progress = 1)) //Charging items like this is slower because I know someone will bite my head off otherwise.
+			induce(C)
+			user.visible_message("[user] recharged the [W]!",\
+								 "<span class='notice'>You recharged the [W]!</span>")
+			return
+
+	..()
+
+/obj/item/weapon/inducer/attack(mob/M, mob/user)
+	if(!user.IsAdvancedToolUser())
+		to_chat(user, "<span class='warning'>You don't have the dexterity to use \the [src]!</span>")
+		return
+
+	if(!cell)
+		to_chat(user, "<span class='warning'>The [src] doesn't have a power cell installed!</span>")
+		return
+
+	if(!cell.charge)
+		to_chat(user, "<span class='warning'>The [src]'s battery is dead!</span>")
+		return
+
+	var/obj/item/weapon/stock_parts/cell/C = M.get_cell()
+	if(C)
+		if(C.charge == C.maxcharge)
+			to_chat(user, "<span class='notice'>[M] is fully charged!</span>")
+			return
+
+		user.visible_message("[user] starts recharging [M] with \the [src]",\
+							"<span class='notice'>You start recharging [M] with \the [src]</span>")
+		if(do_after(user, 10, target = M, progress = 1)) //Charging borgs is fast, since they can have high capacity cells inside.
+			induce(C)
+			user.visible_message("[user] recharged [M]!",\
+								 "<span class='notice'>You recharged [M]!</span>")
+		return
+	..()
+
+
+/obj/item/weapon/inducer/attack_hand(mob/user)
+	if(usr == user && opened && (!issilicon(user)) && user.is_holding(src))
+		if(cell)
+			user.put_in_hands(cell)
+			cell.add_fingerprint(user)
+			cell.updateicon()
+			src.cell = null
+			update_icon()
+			user.visible_message("[user] removes the power cell from [src]!",\
+								  "<span class='notice'>You remove the power cell.</span>")
+	else
+		..()
+
+
+/obj/item/weapon/inducer/examine(mob/living/M)
+	..()
+	if(cell)
+		to_chat(M, "<span class='notice'>The [src]'s display shows: [src.cell.charge]W</span>")
+	else
+		to_chat(M,"<span class='notice'>The [src]'s display is dark.</span>")
+	if(opened)
+		to_chat(M,"<span class='notice'>The [src]'s battery compartment is open.</span>")
+
+/obj/item/weapon/inducer/update_icon()
+	cut_overlays()
+	update_overlays()
+
+/obj/item/weapon/inducer/proc/update_overlays()
+	if(!opened)
+		return
+	else if(!cell)
+		add_overlay("inducer-nobat")
+	else
+		add_overlay("inducer-bat")
+	if(blood_DNA)
+		add_blood(blood_DNA)
+
+
+/obj/item/weapon/inducer/sci
+	icon_state = "inducer-sci"
+	item_state = "inducer-sci"
+
+/obj/item/weapon/inducer/sci/Initialize()
+	. = ..()
+	desc += " This one has a science color scheme."
```

Comment:
You are probably better off just writing a new description for the item under its definition, instead of appending additional text when it is created.